packages/core/src/browser/shell/description-widget.ts

```diff
@@ -0,0 +1,29 @@
+/********************************************************************************
+ * Copyright (C) 2017 TypeFox and others.
```

Comment:
As you created this file yourself, please change the copyright notice to you/your company.